apps/wallet-mobile/src/TxHistory/ActionsBanner.tsx

```diff
@@ -164,10 +173,26 @@ const styles = StyleSheet.create({
     fontWeight: '500',
     lineHeight: 18,
   },
-
   disabled: {
     opacity: 0.5,
   },
+  isCopying: {
+    position: 'absolute',
+    backgroundColor: '#000',
```

Comment:
no hardcoded color should be present in the styles - all should come from the theme package